bin/java-petstore-jersey2.json

```diff
@@ -1,4 +1,7 @@
 {
     "library": "jersey2",
-    "artifactId": "petstore-jersey2"
+    "artifactId": "petstore-jersey2",
+	"additionalProperties" : {
+		"useNullForUnknownEnumValue" : true
```

Comment:
Minor suggestion: What about using spaces instead of tabs?

Reply:
Implemented. Thank you for the review.